crypto/Cargo.toml

```diff
@@ -102,7 +102,7 @@ fluvio-wasm-timer = "0.2"
 time = { version = "0.3", features = ["wasm-bindgen"] }
 
 [dev-dependencies.core-crypto-keystore]
-version = "^1.0.0-rc.60"
+version = "^1.0.0"
```

Comment:
can you move this to the workspace dependency also?

Reply:
Done.